jobsdb/integration_test.go

```diff
@@ -789,100 +789,172 @@ func TestJobsDB_IncompatiblePayload(t *testing.T) {
 	})
 }
 
+// BenchmarkJobsdb takes time... keep waiting
 func BenchmarkJobsdb(b *testing.B) {
+
+	// We are intentionally not using b.N, since we want to have a testbench for stress testing jobsdb's behaviour for longer periods of time (5-15 seconds)
+	const (
+		// totalJobs is the total number of jobs we want our workers to send to the jobsdb, regardless of concurrency
+		totalJobs = 10000
+		// maxDsSize is the maximum ds size, controls how often a new DS will be created
+		maxDsSize = 500
+		// pageSize is the batch size for appending and retrieving jobs within each worker
+		pageSize = 10
+	)
+
+	concurrencies := []int{16, 64, 256, 512}
+
+	os.Setenv("RSERVER_JOBS_DB_MAX_DSSIZE", fmt.Sprintf("%d", maxDsSize))
```

Comment:
Prefer https://pkg.go.dev/testing#B.Setenv 
> Setenv calls os.Setenv(key, value) and uses Cleanup to restore the environment variable to its original value after the test.


```suggestion
	b.Setenv("RSERVER_JOBS_DB_MAX_DSSIZE", fmt.Sprintf("%d", maxDsSize))
```

Reply:
FYI this cannot be used in prallel tests.

Reply:
yes, it is a benchmark so shouldn't be an issue